x11-packages/kf6-kidletime/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE="https://invent.kde.org/frameworks/kidletime"
+TERMUX_PKG_DESCRIPTION="Monitoring user activity"
```

Comment:
I think the more common description fits better here.
https://repology.org/project/kidletime/information
```suggestion
TERMUX_PKG_DESCRIPTION="Reporting of idle time of user and system"
```